### PR TITLE
Fix dangling constructor reference in deprecation note

### PR DESCRIPTION
**Pull request by JarvisCraft**
# Description

This replaces dangling Javadoc reference with the correct one in `PostgreSQLContainer()` deprecation note.

### CODE REVIEW

File: modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     private static final String QUERY_PARAM_SEPARATOR = "&";
 
     /**
-     * @deprecated use {@link PostgreSQLContainer(DockerImageName)} instead
+     * @deprecated use {@link PostgreSQLContainer(String)} instead
```
Comment (rnorth) [inline modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java:38]: Actually I see the problem, but the provided change also suffers from it.

```suggestion
     * @deprecated use {@link PostgreSQLContainer#PostgreSQLContainer(DockerImageName)} or 
     * {@link PostgreSQLContainer#PostgreSQLContainer(String)} instead
```

I think it needs to be like this, adding `#PostgreSQLContainer` to clarify that this is a constructor reference. Also, we may as well point to both the available non-deprecated constructors.

**Review by rnorth — CHANGES_REQUESTED** [review]

File: modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java (hunk 1 above)
Comment (JarvisCraft) [reply modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java]: Right; I've had it fixed (ammended this change to the commit).

Comment (rnorth) [reply modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java]: Thanks! 😄

**Review by rnorth — APPROVED** [review]
Thanks @JarvisCraft

**Review by kiview — APPROVED** [review]
Thanks for contribution @JarvisCraft.

**Pull request merged**